_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+for increasing block size because preventing old nodes from seeing the
+transactions in the extension block also prevents them from being able
+to enforce any other consensus rules on those transactions---such as
+rules that prevent a malicious user from spending other users' bitcoins
+or from creating more bitcoins than allowed by the 21 million bitcoin
+subsidy schedule.
+
+However, one doesn't need to increase block size to increase the amount
+of data that can be added to the block chain per minute---it's also
+possible to increase capacity by increasing the frequency of blocks
+(reducing the average time between blocks).  A method for gaming
+Bitcoin's difficulty adjustment algorithm---called a time-warp
+attack---is well-known among experts and has been used successfully in
+demonstration attacks against Bitcoin's testnet and real attacks against
+altcoins.  (Note: although Bitcoin is technically vulnerable to this
+attack, it'd be a slow attack that would give the userbase a significant
+amount of time to respond.)  By itself, increasing block frequency is
+also an undesirable method for increasing capacity because shorter block
+intervals increase the effectiveness of miners with large amounts of
+hashrate and so is likely to increase mining centralization.[^freq-pow-waste]
+
+Perhaps disproving the saying that "two wrongs don't make a right," this
+talk describes a novel way of combining extension blocks and the
+time-warp attack to allow both upgraded nodes and old nodes to gain the
+same capacity increase and see all the same transactions for validation
+while simultaneously slightly reducing mining centralization risk.
+Upgraded nodes would validate one or more extension blocks providing
+additional block space with a 15 minute average interval, but they'd
+also restrict the time stamps in legacy blocks to ensure a permanent
+(but limited) time warp attack increased the frequency of legacy blocks
+enough to allow them to include the same transactions that previously
+appeared in the extension blocks.
+
+*[video][vid forward blocks], [transcript][tx forward blocks],
+[paper][paper forward blocks], soft fork*
+
+### Compact multi-signatures for smaller blockchains
+
+*Research by Dan Boneh, Manu Drijvers and Gregory Neven*
+
+This talk describes an alternative to the Bellare-Neven-based Schnorr
+signature scheme described in the [MuSig paper][] and [proposed as a
+standard][schnorr pre-bip] for Bitcoin by Pieter Wuille.  The
+alternative method makes use of [pairing-based cryptography][],
+specifically an adaptation of the [Boneh-Lynn-Shacham (BLS) signature
+scheme][bls sigs].  Although pairing-based schemes require an additional
+fundamental security assumption beyond those made by both Bitcoin's
+current ECDSA scheme and proposed Schnorr scheme, the authors present
+evidence that their scheme would produce smaller signatures in general,
+be much more computationally efficient in some multisignature use cases,
```

Comment:
I don't think 'be much more computationally efficient' is accurate. The presented scheme requires pairing operations. Even if a single pairing operation can be shared across multiple signature verifications, it is still some order(s) of magnitude slower than simple EC multiplications.

Reply:
Pieter has confirmed this. The BLS system always requires more computation because the pairing operation is more expensive. The system demonstrated by Neven in this talk reduces the number of pairing operations across aggregated signatures, but there always need to be at least two (one for the signatures/generator and one for the messages/pubkeys).

The advantage of the BLS system is that aggregation can be non-interactive.